src/Sarif.Viewer.VisualStudio.Core/Telemetry/ITelemetryProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.Sarif.Viewer.Telemetry
+{
```

Comment:
Please add doc comments along with <inheritdoc .../> in the implementation class

Reply:
add comments